cli.js

```diff
@@ -121,6 +121,12 @@ program
 
 							writeStream.write((wroteData ? '\n' : '') + data.join('\n'));
 							break;
+
+						case 'stdout':
+							data = _.map(data, function(row) {
+								console.log(row.ipAddress + ':' + row.port);
+							});
```

Comment:
Makes sense, also in case the implementation is changed - will adapt it accordingly.